2012/06/17 13:49:09 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in /home/koleto/public_html/calendar/framework/web/CWebApplication.php:280
Stack trace:
#0 /home/koleto/public_html/calendar/framework/web/CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 /home/koleto/public_html/calendar/framework/base/CApplication.php(162): CWebApplication->processRequest()
#2 /home/koleto/public_html/calendar/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/06/17 13:49:10 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in /home/koleto/public_html/calendar/framework/web/CWebApplication.php:280
Stack trace:
#0 /home/koleto/public_html/calendar/framework/web/CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 /home/koleto/public_html/calendar/framework/base/CApplication.php(162): CWebApplication->processRequest()
#2 /home/koleto/public_html/calendar/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/06/17 13:49:44 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in /home/koleto/public_html/calendar/framework/web/CWebApplication.php:280
Stack trace:
#0 /home/koleto/public_html/calendar/framework/web/CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 /home/koleto/public_html/calendar/framework/base/CApplication.php(162): CWebApplication->processRequest()
#2 /home/koleto/public_html/calendar/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/06/17 13:49:45 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in /home/koleto/public_html/calendar/framework/web/CWebApplication.php:280
Stack trace:
#0 /home/koleto/public_html/calendar/framework/web/CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 /home/koleto/public_html/calendar/framework/base/CApplication.php(162): CWebApplication->processRequest()
#2 /home/koleto/public_html/calendar/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/06/17 13:49:47 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in /home/koleto/public_html/calendar/framework/web/CWebApplication.php:280
Stack trace:
#0 /home/koleto/public_html/calendar/framework/web/CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 /home/koleto/public_html/calendar/framework/base/CApplication.php(162): CWebApplication->processRequest()
#2 /home/koleto/public_html/calendar/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/06/17 13:49:49 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in /home/koleto/public_html/calendar/framework/web/CWebApplication.php:280
Stack trace:
#0 /home/koleto/public_html/calendar/framework/web/CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 /home/koleto/public_html/calendar/framework/base/CApplication.php(162): CWebApplication->processRequest()
#2 /home/koleto/public_html/calendar/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/06/17 13:49:50 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in /home/koleto/public_html/calendar/framework/web/CWebApplication.php:280
Stack trace:
#0 /home/koleto/public_html/calendar/framework/web/CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 /home/koleto/public_html/calendar/framework/base/CApplication.php(162): CWebApplication->processRequest()
#2 /home/koleto/public_html/calendar/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/06/17 13:49:51 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in /home/koleto/public_html/calendar/framework/web/CWebApplication.php:280
Stack trace:
#0 /home/koleto/public_html/calendar/framework/web/CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 /home/koleto/public_html/calendar/framework/base/CApplication.php(162): CWebApplication->processRequest()
#2 /home/koleto/public_html/calendar/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/06/17 13:49:53 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in /home/koleto/public_html/calendar/framework/web/CWebApplication.php:280
Stack trace:
#0 /home/koleto/public_html/calendar/framework/web/CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 /home/koleto/public_html/calendar/framework/base/CApplication.php(162): CWebApplication->processRequest()
#2 /home/koleto/public_html/calendar/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/06/17 13:49:53 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in /home/koleto/public_html/calendar/framework/web/CWebApplication.php:280
Stack trace:
#0 /home/koleto/public_html/calendar/framework/web/CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 /home/koleto/public_html/calendar/framework/base/CApplication.php(162): CWebApplication->processRequest()
#2 /home/koleto/public_html/calendar/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/06/17 13:49:53 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in /home/koleto/public_html/calendar/framework/web/CWebApplication.php:280
Stack trace:
#0 /home/koleto/public_html/calendar/framework/web/CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 /home/koleto/public_html/calendar/framework/base/CApplication.php(162): CWebApplication->processRequest()
#2 /home/koleto/public_html/calendar/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/06/17 13:49:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in /home/koleto/public_html/calendar/framework/web/CWebApplication.php:280
Stack trace:
#0 /home/koleto/public_html/calendar/framework/web/CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 /home/koleto/public_html/calendar/framework/base/CApplication.php(162): CWebApplication->processRequest()
#2 /home/koleto/public_html/calendar/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/06/17 13:00:35 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'calendar'' in C:\wamp\www\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\wamp\www\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\wamp\www\framework\db\CDbConnection.php(309): CDbConnection->setActive(true)
#2 C:\wamp\www\framework\base\CModule.php(388): CDbConnection->init()
#3 C:\wamp\www\framework\base\CApplication.php(431): CModule->getComponent('db')
#4 C:\wamp\www\framework\db\ar\CActiveRecord.php(615): CApplication->getDb()
#5 C:\wamp\www\framework\db\ar\CActiveRecord.php(2263): CActiveRecord->getDbConnection()
#6 C:\wamp\www\framework\db\ar\CActiveRecord.php(379): CActiveRecordMetaData->__construct(Object(User))
#7 C:\wamp\www\protected\modules\user\models\User.php(33): CActiveRecord::model('User')
#8 C:\wamp\www\protected\modules\user\components\UserIdentity.php(27): User::model()
#9 C:\wamp\www\protected\modules\user\models\UserLogin.php(52): UserIdentity->authenticate()
#10 C:\wamp\www\framework\validators\CInlineValidator.php(43): UserLogin->authenticate('password', Array)
#11 C:\wamp\www\framework\validators\CValidator.php(197): CInlineValidator->validateAttribute(Object(UserLogin), 'password')
#12 C:\wamp\www\framework\base\CModel.php(158): CValidator->validate(Object(UserLogin), NULL)
#13 C:\wamp\www\protected\modules\user\controllers\LoginController.php(19): CModel->validate()
#14 C:\wamp\www\framework\web\actions\CInlineAction.php(50): LoginController->actionLogin()
#15 C:\wamp\www\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#16 C:\wamp\www\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#17 C:\wamp\www\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 C:\wamp\www\framework\web\CWebApplication.php(276): CController->run('')
#19 C:\wamp\www\framework\web\CWebApplication.php(135): CWebApplication->runController('user/login')
#20 C:\wamp\www\framework\base\CApplication.php(162): CWebApplication->processRequest()
#21 C:\wamp\www\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/user/login
HTTP_REFERER=http://localhost/user/login
---
2012/06/17 14:06:01 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'calendar'' in C:\wamp\www\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\wamp\www\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\wamp\www\framework\db\CDbConnection.php(309): CDbConnection->setActive(true)
#2 C:\wamp\www\framework\base\CModule.php(388): CDbConnection->init()
#3 C:\wamp\www\framework\base\CApplication.php(431): CModule->getComponent('db')
#4 C:\wamp\www\framework\db\ar\CActiveRecord.php(615): CApplication->getDb()
#5 C:\wamp\www\framework\db\ar\CActiveRecord.php(2263): CActiveRecord->getDbConnection()
#6 C:\wamp\www\framework\db\ar\CActiveRecord.php(379): CActiveRecordMetaData->__construct(Object(User))
#7 C:\wamp\www\protected\modules\user\models\User.php(33): CActiveRecord::model('User')
#8 C:\wamp\www\protected\modules\user\components\UserIdentity.php(27): User::model()
#9 C:\wamp\www\protected\modules\user\models\UserLogin.php(52): UserIdentity->authenticate()
#10 C:\wamp\www\framework\validators\CInlineValidator.php(43): UserLogin->authenticate('password', Array)
#11 C:\wamp\www\framework\validators\CValidator.php(197): CInlineValidator->validateAttribute(Object(UserLogin), 'password')
#12 C:\wamp\www\framework\base\CModel.php(158): CValidator->validate(Object(UserLogin), NULL)
#13 C:\wamp\www\protected\modules\user\controllers\LoginController.php(19): CModel->validate()
#14 C:\wamp\www\framework\web\actions\CInlineAction.php(50): LoginController->actionLogin()
#15 C:\wamp\www\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#16 C:\wamp\www\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#17 C:\wamp\www\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 C:\wamp\www\framework\web\CWebApplication.php(276): CController->run('')
#19 C:\wamp\www\framework\web\CWebApplication.php(135): CWebApplication->runController('user/login')
#20 C:\wamp\www\framework\base\CApplication.php(162): CWebApplication->processRequest()
#21 C:\wamp\www\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/user/login
HTTP_REFERER=http://localhost/user/login
---
2012/06/17 14:06:03 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'calendar'' in C:\wamp\www\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\wamp\www\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\wamp\www\framework\db\CDbConnection.php(309): CDbConnection->setActive(true)
#2 C:\wamp\www\framework\base\CModule.php(388): CDbConnection->init()
#3 C:\wamp\www\framework\base\CApplication.php(431): CModule->getComponent('db')
#4 C:\wamp\www\framework\db\ar\CActiveRecord.php(615): CApplication->getDb()
#5 C:\wamp\www\framework\db\ar\CActiveRecord.php(2263): CActiveRecord->getDbConnection()
#6 C:\wamp\www\framework\db\ar\CActiveRecord.php(379): CActiveRecordMetaData->__construct(Object(User))
#7 C:\wamp\www\protected\modules\user\models\User.php(33): CActiveRecord::model('User')
#8 C:\wamp\www\protected\modules\user\components\UserIdentity.php(27): User::model()
#9 C:\wamp\www\protected\modules\user\models\UserLogin.php(52): UserIdentity->authenticate()
#10 C:\wamp\www\framework\validators\CInlineValidator.php(43): UserLogin->authenticate('password', Array)
#11 C:\wamp\www\framework\validators\CValidator.php(197): CInlineValidator->validateAttribute(Object(UserLogin), 'password')
#12 C:\wamp\www\framework\base\CModel.php(158): CValidator->validate(Object(UserLogin), NULL)
#13 C:\wamp\www\protected\modules\user\controllers\LoginController.php(19): CModel->validate()
#14 C:\wamp\www\framework\web\actions\CInlineAction.php(50): LoginController->actionLogin()
#15 C:\wamp\www\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#16 C:\wamp\www\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#17 C:\wamp\www\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 C:\wamp\www\framework\web\CWebApplication.php(276): CController->run('')
#19 C:\wamp\www\framework\web\CWebApplication.php(135): CWebApplication->runController('user/login')
#20 C:\wamp\www\framework\base\CApplication.php(162): CWebApplication->processRequest()
#21 C:\wamp\www\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/user/login
HTTP_REFERER=http://localhost/user/login
---
